tests/managed/interfaces.cs

```diff
@@ -6,9 +6,9 @@ public interface IMakeItUp {
 
 		bool Boolean { get; }
 
-		string Convert (int integer);
+		string Convert (int value);
 
-		string Convert (long longint);
+		string Convert (long value);
```

Comment:
The test was trying (fwict) to verify what happens when you have a collision. However, the improved detection code won't detect a collisions as it will generate names w\ the first argument included.

I can remove this and fixup the test if you don't believe it was intended.